code/modules/projectiles/guns/misc/beam_rifle.dm

```diff
@@ -509,6 +509,8 @@
 		return 0.4
 	if(istype(target, /obj/structure/window))
 		return 0.5
+	if(istype(target, /obj/structure/blob))

```

Comment:
if you aren't sending this to upstream, add
`//CIT CHANGE - removes an easy ggnore method of dealing with blobs`
here